2.4/contrib/httpd_util.sh

```diff
@@ -0,0 +1,14 @@
+# source
```

Comment:
I think this is pretty ugly. Requiring people that use this image as a base layer to 'source' some script is not what I think we should be doing.

The better approach would be to rename/replace the default ENTRYPOINT in the latter image (you can preserve it and call it from your ENTRYPOINT or you can just override it).